### ISSUE TITLE
Claude Dev stuck in loop, dont see any activity

### ISSUE THREAD

**Issue by UtsavG123** [clinebot/cline#418]
### Which API Provider are you using?

Google Gemini

### Which Model are you using?

gemini-1.5-flash-002

### What happened?

![image](https://github.com/user-attachments/assets/55453895-5096-44d5-83c1-756141c758b6)
it gets stuck in this loop with no way out seemingly. 0 tokens up or down.
other AI models seem to be working fine, seems like a gemini specific issue.
more intriguingly, i tried using my gemini api key with claude dev on another machine and there it seemed to work fine.
in short, here is what it seems like - seems to be an issue with using gemini api and on my machine
is there some way to get out of this? are there any steps to do clean reinstall or vs code, claude dev - like just reset everything to go back to normal?

### Steps to reproduce

Noob here, so pardon the retarded note - 
1.just set up a gemini with api key and choose a model
2. prompt something to claude dev
3. it gets stuck

**Comment by saoudrizwan** [comment]
v2.0 brings better performance for weaker models like gemini flash, please give it a try!
